packages/expo-modules-core/ios/Swift/Modules/AnyModule.swift

```diff
@@ -0,0 +1,54 @@
+
+/**
+ A protocol for any type-erased module that provides methods used by the core.
```

Comment:
```suggestion
 A protocol for any type-erased module that provides methods used by the core.
```
What is the meaning of `type-erased module` - mind adding line of explanation? 🤔

Reply:
I don't think we should explain what common Swift terms mean. You can always google it 😄

When you work with generics like the `ConcreteMethod<...>`, you cannot actually have an array of type `[ConcreteMethod]` nor `[ConcreteMethod<Any...>]`, so we need something that "hides" the generics so we can do `[AnyMethod]` instead.